packager/media/formats/mp2t/mpeg1_header.h

```diff
@@ -7,6 +7,11 @@
 
 #include "packager/media/formats/mp2t/audio_header.h"
 
+namespace {
+const uint8_t kMpeg1V_INV = 0b01; /* Invalid version */
+const uint8_t kMpeg1L_INV = 0b00; /* Invalid layer */
+}  // namespace
```

Comment:
[Google C++ style guide](https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables) disallows having unnamed namespace in header files. I think it is fine to just default initialize them to 0 as what we want to prevent is undefined behaviors.